pfcpiface/telemetry.go

```diff
@@ -89,5 +129,52 @@ func (uc *upfCollector) summaryLatencyJitter(ch chan<- prometheus.Metric) {
 func setupProm(upf *upf) {
 	uc := newUpfCollector(upf)
 	prometheus.MustRegister(uc)
+	if err := prometheus.Register(pfcpEstReq); err != nil {
+		log.Println("pfcpEstReq not registered:", err)
+	} else {
+		log.Println("pfcpEstReq registered.")
+	}
+
+	if err := prometheus.Register(pfcpModReq); err != nil {
+		log.Println("pfcpModReq not registered:", err)
+	} else {
+		log.Println("pfcpModReq registered.")
+	}
+
+	if err := prometheus.Register(pfcpDelReq); err != nil {
+		log.Println("pfcpDelReq not registered:", err)
+	} else {
+		log.Println("pfcpDelReq registered.")
+	}
+	if err := prometheus.Register(pfcpAssocReq); err != nil {
+		log.Println("pfcpAssocReq not registered:", err)
+	} else {
+		log.Println("pfcpAssocReq registered.")
+	}
+	if err := prometheus.Register(pfcpEstRsp); err != nil {
+		log.Println("pfcpEstRsp not registered:", err)
+	} else {
+		log.Println("pfcpEstRsp registered.")
+	}
+	if err := prometheus.Register(pfcpModRsp); err != nil {
+		log.Println("pfcpModRsp not registered:", err)
+	} else {
+		log.Println("pfcpModRsp registered.")
+	}
+	if err := prometheus.Register(pfcpDelRsp); err != nil {
+		log.Println("pfcpDelRsp not registered:", err)
+	} else {
+		log.Println("pfcpDelRsp registered.")
+	}
+	if err := prometheus.Register(pfcpAssocRsp); err != nil {
+		log.Println("pfcpAssocRsp not registered:", err)
+	} else {
+		log.Println("pfcpAssocRsp registered.")
+	}
+	if err := prometheus.Register(activeUsersCnt); err != nil {
+		log.Println("activeUsersCnt not registered:", err)
+	} else {
+		log.Println("activeUsersCnt registered.")
+	}
```

Comment:
Refactor the vars above and these lines into a `pfcpCollector` ? similar to `upfCollector` ?

Reply:
using one vector for all pfcp messages and registering only that. so don't need to use a collector now I think.